crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -1112,4 +1144,16 @@ mod test {
             "nonce should have been deleted also"
         );
     }
+
+    #[test]
+    fn snapshots() {
```

Comment:
nit: more descriptive test name perhaps `storage_keys_have_not_changed` or something like that?

Reply:
Good idea. Done for this and the other snapshot tests in  5fd2ae5